Sources/App/Views/PackageController/Builds/BuildIndex+Model.swift

```diff
@@ -37,14 +37,12 @@ extension BuildIndex {
         }
 
         init?(packageInfo: PackageController.BuildsRoute.PackageInfo, buildInfo: [PackageController.BuildsRoute.BuildInfo]) {
-            guard let packageName = packageInfo.packageName else { return nil }
-
```

Comment:
I kept existing behaviour here when I touch this last week but my intention was to fill it in with the owner here instead of letting it trickle into the views (and keep `packageName: String` instead of `String?`. I.e.

```swift
            self.init(owner: packageInfo.repositoryOwner,
                      repositoryName: packageInfo.repositoryName,
                      packageName: packageInfo.packageName ?? packageInfo.repositoryName,
                      buildGroups: buildGroups)
```

This would then also make the initialiser non-optional etc and all that.